tests/test_span_session.rb

```diff
@@ -0,0 +1,99 @@
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/span_session'
+require_relative '../lib/cisco_node_utils/interface'
+require_relative '../lib/cisco_node_utils/vlan'
+require_relative '../lib/cisco_node_utils/cisco_cmn_utils'
+
+include Cisco
+
+# TestSpanSession - Minitest for SPAN session node utility
+class TestSpanSession < CiscoTestCase
+  @skip_unless_supported = 'session'
```

Comment:
This line needs to match the name of the yaml file.  Otherwise it will always skip the tests.

 `@skip_unless_supported = 'span_session'`